Dockerfile

```diff
@@ -1,30 +1,35 @@
-FROM node:5.0
-MAINTAINER Daniel Schaefer <daniel.schaefer@hpe.com>
+# This project is currently built against MeteorJS 1.4.2.7, which bundles
+# Node.js 4.4.7. The most recent and compatible Node.js image was picked to
+# provide the most compoatible build tooling for `meteor npm install'.
+FROM node:4.9.1-stretch as builder
```

Comment:
May I know why we choose the node:4.9.1-stretch instead of 5.0

Reply:
Read me comment above.